map.js

```diff
@@ -43,33 +165,28 @@ function placePoint(coords) {
 function initializeMap(clickHandler) { // eslint-disable-line no-unused-vars
   $('body').addClass('mapped');
   $('#map').children().remove();
+  // return;
+  var centerPoint = new Coordinate(0, 0);
 
   olMap = new ol.Map({
     target: 'map',
     layers: [
       new ol.layer.Tile({
-        source: new ol.source.OSM()
-      }),
-      new ol.layer.Vector({
-      }),
+        source: new ol.source.OSM(),
+        type: 'tile',
+      })
     ],
     view: new ol.View({
-      center: ol.proj.fromLonLat([0, 0]),
+      center: centerPoint.projection,
       zoom: 1
     })
   });
 
   olMap.on(
     'click',
     function(event) {
-      var coordArray = ol.proj.transform(
-        event.coordinate,
-        'EPSG:3857',
-        'EPSG:4326'
-      );
-      var coordObj = {lat: coordArray[1], lng: coordArray[0]};
-      placePoint(coordObj);
-      clickHandler(coordObj);
+      var coord = new Coordinate(event.coordinate);
+      clickHandler(coord);
     }
   );
 }
```

Comment:
Good work on this. I think this is the most encapsulated way I've seen OpenLayers be used- use of the constructor for coordinate points. :sparkles: :+1: :sparkles: